File: 66_cap_so_tong_bang_K.py
def solve():
    global  n  , M
    sum = M[0] + M[1]
    for i in range(0 , n-1):
        for j in range(i+1, n):
            if abs(M[i] + M[j]) < abs(sum):
                sum = M[i] + M[j]
    return sum
t = int(input())
while t>0:
    n  = int(input())
    M = list(map(int, input().split()))
    print((solve()))
    t-=1

File: 66_cap_so_tong_bang_K.cpp
#include<iostream>
using namespace std ; 

void solve() {
    int n  , k ;cin>> n >>k  ; int M[n] ; 
    for(int i = 0 ; i<n  ; i++) cin>>M[i] ; 
    int dem = 0 , sum ; 
    for(int i= 0 ; i<n-1 ; i++) {
        for(int j=i+1 ; j<n ; j++) {
            sum = M[i]+M[j] ; 
            if(sum == k ) dem++ ; 
        }
    }
    cout<<dem<<endl ;
}
main() {
    int t; cin>>t ;
    while(t--) {
        solve() ; 
    }
}